src/protocol/fluvio-protocol-core/src/encoder.rs

```diff
@@ -147,6 +161,32 @@ where
     }
 }
 
+/// Ignore encoding for raw bytes.
+impl Encoder for Bytes {
```

Comment:
there should not be special treatment for bytes

Reply:
This was added just in order to allow the sending of raw data (used for partial sends). For that case `fluvio_frame_bytes` should not append the length at the beginning.